src/Common/tests/Scripts/Tools/net_startlog.cmd

```diff
@@ -1,2 +1,2 @@
-logman create trace SystemNetTrace -o net.etl -pf net_providers.txt
+logman create trace SystemNetTrace -o net.etl -pf net_providers.txt -max 1000
```

Comment:
Why 1000 MB? It doesn't sounds unreasonable, just want to know that this is deliberate.

Reply:
These scripts are really just "samples" for someone to use or not use.  It was not obvious to me why PerfView was giving an error message about traces being dropped when I opened up a networking.etl trace file.

Past sample scripts we have used internally have used this parameter value. So, it seems reasonable as a guide.  Anyone can choose to not use this sample script and/or do it differently.